cognite/client/_api/datapoints.py

```diff
@@ -1231,7 +1235,9 @@ def retrieve_latest(
                 ...     id=id_queries,
                 ...     external_id=LatestDatapointQuery(external_id="abc", before="3h-ago"))
         """
-        fetcher = RetrieveLatestDpsFetcher(id, external_id, before, ignore_unknown_ids, self)
+        fetcher = RetrieveLatestDpsFetcher(
```

Comment:
Let's add an example to the docstring showcasing how to specify unit 🚀

Reply:
Done